webextensions/api/tabs.json

```diff
@@ -3792,7 +3785,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/tabs/sendMessage",
             "support": {
               "chrome": {
-                "version_added": "≤41"
+                "version_added": "20"
```

Comment:
✅ https://github.com/chromium/chromium/commit/00536215259ebc2e3c82347e3812770c1a89f03d in [20.0.1090.0](https://github.com/chromium/chromium/releases/tag/20.0.1090.0) before 20.0.1132.

ℹ️ Previously `sendRequest` since https://github.com/chromium/chromium/commit/5e34bc403af2328e80765406eace2f49f63e7d79 in [4.0.229.1](https://github.com/chromium/chromium/releases/tag/4.0.229.1) AFTER 4.0.223 (i.e. Chrome 5).